Products/GenericSetup/components.py

```diff
@@ -22,11 +24,7 @@
 from zope.component import queryMultiAdapter
 from zope.component.interfaces import ComponentLookupError
 from zope.component.interfaces import IComponentRegistry
-try:
-    from zope.component.interfaces import IPossibleSite
-except ImportError:
-    # BBB: for Zope < 2.13 (zope.component < 3.8)
-    from zope.location.interfaces import IPossibleSite
+from zope.component.interfaces import IPossibleSite
```

Comment:
Do we need to adjust pins in `setup.py`?

Reply:
It already says `Zope2 >= 4.0a6` and it looks like we've punted on listing dependencies that are part of the Zope KGS. Not sure if that's intentional or an oversight?